compile.py

```diff
@@ -116,6 +120,15 @@ def processNavigation( lessonname, actions, embeds, plumeds_to_use,plumed_versio
                     new_name += spl_name[i] + "/"
                  name = new_name + "GAT_SAFE_README.md"
                  shutil.copyfile("data/" + old_name, "data/" + name)
+              nspl_name = name.split("/")
+              if len(nspl_name)>1 ) : 
```

Comment:
```suggestion
              if len(nspl_name)>1 : 
```
this was blocking the CI

Reply:
Fixed that thanks